src/main/webapp/pages/projects/settings/pages/associated.html

```diff
@@ -8,7 +8,7 @@
     <title th:text="#{project.associated.heading(${project.label})}">
       ASSOCIATED PROJECTS
     </title>
-    <link rel="stylesheet" href="/dist/css/associated-projects.bundle.css" />
+    <link rel="stylesheet" href="#" th:href="@{/dist/css/associated-projects.bundle.css}" />
```

Comment:
just remove the regular \`href\` all together, it does nothing.

Reply:
Sure.  I remember for some elements it fails validation if you don't have the non `th:` version of the tag, but I don't think it matters for a `<link>`.

Removed in c30af45.